Sources/DiscoverKit/View/RecommendationsView.swift

```diff
@@ -14,6 +14,11 @@ struct RecommendationsView: View {
     /// Max readable width comes from our investigation we did in this PR: https://github.com/Pocket/pocket-ios/pull/763
     static let maxReadableWidth: CGFloat = 700
 
+    init(recommendations: [Recommendation]) {
+        self.recommendations = recommendations
+        UILabel.appearance(whenContainedInInstancesOf: [UINavigationBar.self]).adjustsFontSizeToFitWidth = true
```

Comment:
This is a workaround that I found, but happen to discuss further on how we should account for larger dynamic fonts while using a navigation title. 

Another option I considered is to not use the navigation title, and instead use a normal `Text` view but then we lose the built in capabilities that come with using the navigation modifier

Reply:
I think this is fine for now @cyndichin ! I believe not everything needs to be scaled and we should keep using the navigation title. How does this look with large fonts if we don't scale it?